extension.neon

```diff
@@ -72,6 +72,21 @@ conditionalTags:
 		phpstan.broker.methodsClassReflectionExtension: %doctrine.allCollectionsSelectable%
 
 services:
+	otherMethodAnalysisParser: # identical to defaultAnalysisParser, but uses inner PathRoutingParser without cleaning parser
+		class: PHPStan\Parser\CachedParser
+		arguments:
+			originalParser: @pathRoutingParser
+			cachedNodesByStringCountMax: %cache.nodesByStringCountMax%
+		autowired: false
+
+	pathRoutingOtherMethodAnalysisParser: # identical to pathRoutingParser, but no cleaning parser is used
```

Comment:
I don't get it. Where is this one used?

Reply:
That is obviously a typo, it should be used above. Fixed